pkg/controller/constrainttemplate/constrainttemplate_controller.go

```diff
@@ -180,7 +207,9 @@ func (r *ReconcileConstraintTemplate) handleCreate(
 	}
 	log.Info("creating constraint CRD")
 	if err := r.Create(context.TODO(), crd); err != nil {
-		instance.Status.Error = fmt.Sprintf("Could not create CRD: %s", err)
+		instance.Status.Errors = []*v1alpha1.CreateCRDError{}
+		createErr := &v1alpha1.CreateCRDError{Code: "unknown_error", Message: fmt.Sprintf("Could not create CRD: %s", err), Location: ""}
```

Comment:
This could be a known error

Reply:
I've replaced this with "create_error". Just wanted something for any errors that could come back without a code field.